e2e-tests/IIoTPlatform-E2E-Tests/TestModels/PublishedNodesEntryModel.cs

```diff
@@ -10,6 +10,21 @@ namespace IIoTPlatform_E2E_Tests.TestModels {
     /// Model with nodes to publish per Endpoint
     /// </summary>
     public class PublishedNodesEntryModel {
```

Comment:
I think this class is not part of the API, therefore should not be used in the E2E testing.